### DISCUSSION TITLE
Bed mesh - heightmap "overnight" way different...

### DISCUSSION THREAD

**Discussion by cyaneo** [bassamanator/everything-sovol-sv06#47] [General]
Hello,

overnight the mesh range changes from 0.384mm to 1.195 mm. A few days ago the bed was "flat" but in front left corner (X/Y) there was 0.384mm

![med Mesh](https://github.com/bassamanator/everything-sovol-sv06/assets/8118394/240062d7-36a2-4237-9d35-6b2db80ea834)

How does this happen?

**Comment by bassamanator** [comment]
I imagine that something is loose/not quite as secure as it should be. Run through the initial steps on my everything sv06 repo.

Helpers: [Use Codeblocks](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) | [Explain the shell](https://explainshell.com/) | [OSS Klipper Config](https://github.com/bassamanator/Sovol-SV06-firmware) | [Everything-Sovol-SV06](https://github.com/bassamanator/everything-sovol-sv06) | [Use for pastebin.com for large files](https://pastebin.com/)
